### PR TITLE
Regina REXX Interpreter port

### PR DESCRIPTION
**Pull request by 4mig4**
Please test , the manpages is not registering on whatis db for some reason but I could not figure out why.

### CODE REVIEW

File: lang/regina-rexx/Makefile.inc [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+#SHARED_ONLY=    Yes
+
+CATEGORIES=     lang
+
+MASTER_SITES=  http://jaist.dl.sourceforge.net/sourceforge/regina-rexx/ \
+                http://voxel.dl.sourceforge.net/sourceforge/regina-rexx/ \
+                http://internode.dl.sourceforge.net/sourceforge/regina-rexx/ \
+                http://fastbull.dl.sourceforge.net/sourceforge/regina-rexx/ \
+                http://freefr.dl.sourceforge.net/sourceforge/regina-rexx/
```
Comment (natano) [inline lang/regina-rexx/Makefile.inc]: I think that could be replaced with <code>MASTER_SITES= ${MASTER_SITE_SOURCEFORGE:=regina-rexx/}</code>.

**Comment by jcvernaleo** [comment]
@4mig4 is there a reason you put it in a versioned subdirectory (3.9.0)rather than directly in lang/regina-rexx ?

**Comment by jcvernaleo** [comment]
@4mig4 
jcv@ryu 3.9.0 $ make port-lib-depends-check

regina-rexx-3.9.0(lang/regina-rexx/3.9.0):
Missing: c.78 (/usr/local/bin/rxstack) (system lib)
Missing: m.10 (/usr/local/bin/rexx) (system lib)
Missing: ncurses.15 (/usr/local/bin/rexx) (system lib)
Missing: pthread.20 (/usr/local/bin/rexx) (system lib)
WANTLIB += c m ncurses pthread
*** Error 1 in target 'port-lib-depends-check' (ignored)
Should add the WANTLIB to it that port-lib-depends-check suggests.

**Comment by 4mig4** [comment]
There are more official version of regina-rexx available
I will add them to the repo , namely 3.8.2

**Comment by jcvernaleo** [comment]
Okay, that makes sense.  Since I'm not familiar with regina-rexx I didn't know if that is the case.

**Comment by jcvernaleo** [comment]
As for the man pages, they are being installed:
jcv@ryu man1 $ pwd
/usr/local/man/man1
jcv@ryu man1 $ ls regina*
regina-config.1.gz  regina.1.gz

The problem is they are compressed which is not allowed.  Not sure what magic ports has to deal with that yet though.

**Comment by jcvernaleo** [comment]
@4mig4 One more thing.  Unlike OpenBSD ports we don't really use the MAINTAINER line so could you remove that from the Makefile.

**Comment by 4mig4** [comment]
This port looks like it is fully working now

**Comment by jcvernaleo** [comment]
Thanks for the port!  github didn't feel like closing this automatically so i'm closing it now.

**Pull request closed without merging**